Shared/Samples/Edit features using feature forms/EditFeaturesUsingFeatureFormsView.swift

```diff
@@ -0,0 +1,205 @@
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import ArcGISToolkit
+import SwiftUI
+
+struct EditFeaturesUsingFeatureFormsView: View {
+    /// The map that contains the features for editing.
+    @State private var map: Map = {
+        // Creates the map from the "Feature Form Places" web map portal item.
+        let portalItem = PortalItem(
+            portal: .arcGISOnline(connection: .anonymous),
+            id: .featureFormPlacesWebMap
+        )
+        return Map(item: portalItem)
+    }()
+    
+    /// The feature form for the selected feature.
+    @State private var featureForm: FeatureForm?
+    
+    /// The point on the screen where the user tapped.
+    @State private var tapPoint: CGPoint?
+    
+    /// A Boolean value indicating whether the feature form panel is presented.
+    @State private var isShowingFeatureForm = false
+    
+    /// A Boolean value indicating whether the discard edits alert is presented.
+    @State private var isShowingDiscardEditsAlert = false
+    
+    /// A Boolean value indicating whether the feature form has any validation errors.
+    @State private var hasValidationErrors = false
+    
+    /// A Boolean value indicating whether the feature form edits are being applied.
+    @State private var isApplyingEdits = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The text explaining the next step in the sample's workflow.
+    private var instructionText: String {
+        if !isShowingFeatureForm {
+            "Tap on a feature to edit."
+        } else if hasValidationErrors {
+            "Fix the errors to apply the edits."
+        } else if isApplyingEdits {
+            "Applying edits..."
+        } else {
+            "Use the form to edit the feature's attributes."
+        }
+    }
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map)
+                .onSingleTapGesture { screenPoint, _ in
+                    if isShowingFeatureForm {
+                        isShowingDiscardEditsAlert = true
+                    } else {
+                        tapPoint = screenPoint
+                    }
+                }
+                .task(id: tapPoint) {
+                    // Identifies the feature at the tapped point and creates a feature form from it.
+                    guard let tapPoint,
+                          let featureLayer = map.operationalLayers.first as? FeatureLayer else {
+                        return
+                    }
+                    defer { self.tapPoint = nil }
```

Comment:
This was here to prevent a potential bug where the task wouldn't run if a tap point was the exact same as the previous one (since the `tapPoint` variable wouldn't update). That is pretty unlikely, though, so I removed it.